alf/trainers/on_policy_trainer.py

```diff
@@ -102,6 +115,17 @@ def train_():
             if (iter + 1) % checkpoint_interval == 0:
                 checkpointer.save(global_step=global_step.numpy())
 
+            if (iter + 1) % eval_interval == 0:
+                with tf.summary.record_if(True):
+                    eager_compute(eval_metrics, eval_env,
```

Comment:
Use key=value form, it's less prone to errors in case of future change of the arguments.